addon-test-support/ilios-common/page-objects/components/objective-list-item-terms.js

```diff
@@ -0,0 +1,16 @@
+import { clickable, create, collection, isHidden, isPresent } from 'ember-cli-page-object';
+import selectedTerms from './detail-terms-list';
+
+const definition = {
+  scope: '[data-test-objective-list-item-terms]',
+  list: collection('[data-test-detail-terms-list]', selectedTerms),
+  empty: isHidden('[data-test-parent]'),
```

Comment:
Doesn't seem to be an element on this component.

Reply:
yanked.